### PR TITLE
1511: Improvements from POC

### PR DESCRIPTION
**Pull request by shaunharrisonFR**
Removing need for 7.3.0 in kustomize directory
New secrets for supporting different TLS certs for ingresses
New Ingress name for previously named forgerock ingress

Issue: https://github.com/SecureApiGateway/SecureApiGateway/issues/1511

### CODE REVIEW

File: ob/kustomize/overlay/defaults/kustomization.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ patches:
       metadata:
         name: openig-secrets-env
 resources:
-  - https://github.com/SecureApiGateway/sapig-openbanking-uk-developer-envs/core/kustomize/overlay/7.3.0/defaults?ref=master
+  - https://github.com/SecureApiGateway/sapig-openbanking-uk-developer-envs/core/kustomize/overlay/defaults?ref=1511-template-improvements
```
Comment (dbadham-fr) [inline ob/kustomize/overlay/defaults/kustomization.yaml:38]: I assume you need to update ref once other PR is merged?

Comment (dbadham-fr) [reply ob/kustomize/overlay/defaults/kustomization.yaml:38]: Oh wait, it's referencing this repo?

Comment (shaunharrisonFR) [reply ob/kustomize/overlay/defaults/kustomization.yaml]: Yes that needed to go back to being master, this is where the OB files uses Core as it's base and adds whatever is needed

Comment (dbadham-fr) [reply ob/kustomize/overlay/defaults/kustomization.yaml]: Ah yes, I assume you can't do a relative path then?

Comment (shaunharrisonFR) [reply ob/kustomize/overlay/defaults/kustomization.yaml]: You can, but using it like this to mirror relenvs release repo

**Review by dbadham-fr — APPROVED** [review]

**Pull request merged**